src/TabWidget.js

```diff
@@ -0,0 +1,386 @@
+/**
+ * @class argos.TabWidget
+ * @alternateClassName TabWidget
+ */
+import declare from 'dojo/_base/declare';
+import lang from 'dojo/_base/lang';
+import array from 'dojo/_base/array';
+import domClass from 'dojo/dom-class';
+import domConstruct from 'dojo/dom-construct';
+import domStyle from 'dojo/dom-style';
+import query from 'dojo/query';
+import _Templated from 'argos/_Templated';
+
+const __class = declare('argos.TabWidget', [_Templated], {
+  /**
+   * @property {Simplate}
+   * HTML that starts a new tab list
+   */
+  tabContentTemplate: new Simplate([
+    '{%! $.tabListTemplate %}',
+  ]),
+  /**
+   * @property {Simplate}
+   * HTML that starts a new tab list
+   */
+  tabListTemplate: new Simplate([
+    '<ul class="tab-list" data-dojo-attach-point="tabList">',
+    '</ul>',
+  ]),
+  /**
+   * @property {Simplate}
+   * HTML that starts a new More tab list
+   */
+  moreTabListTemplate: new Simplate([
+    '<ul class="more-tab-dropdown" data-dojo-attach-point="moreTabList">',
+    '</ul>',
+  ]),
+  /**
+   * @property {Simplate}
+   * HTML that starts a new animation bar
```

Comment:
This is really a simplate template that you have to apply to get html, and does it start the animation or defines it?